src/commcare_cloud/ansible/roles/couchdb2/templates/compact_couchdb.sh.j2

```diff
@@ -109,7 +109,7 @@ do
       fi
 
       DEcho "${DB} - Getting view list ...";
-      DESIGNLIST=`CouchReq "GET" "/${DB}/_all_docs?startkey=\"_design/\"&amp;endkey=\"_design0\"&amp;include_docs=true" | awk -F ":" '{print $3}' | awk -F \" '{print $2}' | grep "^_design/" | sed -e s:"_design/"::g`;
+      DESIGNLIST=`CouchReq "GET" "/${DB}/_design_docs|jshon -e rows -a -e id -u|awk -F '/' '{print $2}'`;
```

Comment:
Was the issue here that the line above simply didn't work?

Reply:
Seems , this script has never been in place in any of SaaS environments ever . as default value has been always false. https://github.com/dimagi/commcare-cloud/blob/3a3005834e2c8909898c0ded7887580819726255/src/commcare_cloud/ansible/roles/couchdb2/defaults/main.yml#L13

Reply:
and also looks like the existing line is not producing a valid output which this line can take as input for compaction . https://github.com/dimagi/commcare-cloud/blob/3a3005834e2c8909898c0ded7887580819726255/src/commcare_cloud/ansible/roles/couchdb2/templates/compact_couchdb.sh.j2#L117